expected/15.4/extra/aggregates.out

```diff
@@ -86,6 +86,34 @@ SELECT avg(a) AS avg_32 FROM aggtest WHERE a < 100;
  32.6666666666667
 (1 row)
 
+CREATE FOREIGN TABLE agg_tb(v int, id integer OPTIONS (key 'true')) SERVER sqlite_svr;
+INSERT INTO agg_tb(v) VALUES(1), (2), (3);
+SELECT any_value(v) FROM agg_tb;
+ERROR:  function any_value(integer) does not exist
+LINE 1: SELECT any_value(v) FROM agg_tb;
+               ^
+HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
```

Comment:
It seems be any_value function support from postgres version >= 16.0.
Because you unified the test cases from test version 16.0 to lower versions, please add comments to explain the error message

Reply:
Done, please verify, @MinhLA1410

Reply:
Thank you for supporting. It's OK.

Reply:
Thank you for supporting. It's OK